tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,4 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
```

Comment:
Make sure to use `is` for comparsions against singleton values `False`, `True`, `None`.
```suggestion
    assert result is False
```